### PR TITLE
chore(validation): improve validate example

### PR DESCRIPTION
**Pull request by T-Motion**
Improve validation code example.

### CODE REVIEW

File: packages/validate/README.md [hunk 1]
```diff
@@ -48,11 +48,11 @@ class IsInitialsExample extends Validator {
   }
 
   execute(value) {
```
Comment (jorenbroekema) [inline packages/validate/README.md]: Maybe we could simplify this to:

```js
execute(value) {
  return (!IsString(value) || !isInitialsRegex.test(value.toLowerCase()));
}
```

File: packages/validate/stories/Overview.stories.mdx [hunk 2]
```diff
@@ -52,11 +52,11 @@ class MyValidator extends Validator {
   }
 
   execute(modelValue, param) {
```
Comment (jorenbroekema) [inline packages/validate/stories/Overview.stories.mdx]: Same here, I think

```js
execute() {
  return modelValue === param;
}
```

Is clear enough.

Comment (T-Motion) [reply packages/validate/stories/Overview.stories.mdx]: It should be clear the execute function will show any feedback if it returns true. The way I understood it at first is "when validator returns false, there is an error". This should be clarified imo. The example shouldn't be written efficient I think. It should just explain how it work. What do you think?

Comment (jorenbroekema) [reply packages/validate/stories/Overview.stories.mdx]: For me an example should showcase usage. 

Documentation / types (jsdocs in our case) should annotate what the return value is and what that means in my opinion.

I get your point of view now though and maybe it is indeed nice that the example provides clarity on top of docs. this PR is an improvement for sure. I'm fine with going with your changes ^^ I will edit my review

**Comment by daKmoR** [comment]
nice 👌

**Pull request merged**